src/Parallel/CharmRegistration.hpp

```diff
@@ -356,6 +362,15 @@ struct get_value_type<
   using type = std::pair<Key, Mapped>;
 };
 
+template <size_t Size, class Key, class Mapped, class Hash, class KeyEqual>
+struct get_value_type<FixedHashMap<Size, Key, Mapped, Hash, KeyEqual>> {
+  // When sending data it is typical to use `std:make_pair(a, b)` which results
```

Comment:
`std::` (second colon). 

Here it may also be helpful to rephrase the "which results in a non-const Key type". It sounds like a description of a problem, rather than a description of a desired behavior.
Perhaps:
"which needs a non-const Key type"

If updated, the prior map comment should be updated to be consistent.

Reply:
I am intentionally describing the problem.

Reply:
I expanded the comment. Please let me know if it is clearer.

Reply:
(speaking only for myself) I think the update is a significant improvement.